app/src/main/java/hr/foi/air/servicesync/backend/firebaseAuthentication.kt

```diff
@@ -12,6 +12,8 @@ class firebaseAuthentication {
             .addOnCompleteListener { task ->
                 if (task.isSuccessful) {
                     val user = auth.currentUser
+                    val firesoreUserDetails : FirestoreUserDetails = FirestoreUserDetails()
```

Comment:
Well done decomposing usage of a such function to various layers of the application.